Makefile

```diff
@@ -33,3 +33,13 @@ publish-image: image
 
 minikube-publish: image
 	$(SUDO) docker save docker.io/$(DH_ORG)/kured | (eval $$(minikube docker-env) && docker load)
+
+manifest:
+	sed -i "s#image: docker.io/.*kured.*#image: docker.io/$(DH_ORG)/kured:$(VERSION)#g" kured-ds.yaml
```

Comment:
Side note: I would prefer if our manifests recorded in a specific branch are complete with their version. It allows to build an image, a consuming manifest and helm chart at the same time, and publish them all.